mesheryctl/internal/cli/root/system/start.go

```diff
@@ -79,174 +84,309 @@ func ValidateComposeFileForRecreation(CurrentServices map[string]utils.Service,
 	return nil
 }
 
+// applyManifest is a wrapper function for client.ApplyManifest
+func applyManifest(manifest []byte, client *meshkitkube.Client) error {
+	// ApplyManifest applies the given manifest file to the Kubernetes cluster
+	err := client.ApplyManifest(manifest, meshkitkube.ApplyOptions{
+		Namespace: "default",
+		Update:    true,
+		Delete:    false,
+	})
```

Comment:
@mgfeller That would be best. We can just change the namespace here. So, everything will be deployed to the "meshery" namespace. That works right?

Reply:
> @mgfeller That would be best. We can just change the namespace here. So, everything will be deployed to the "meshery" namespace. That works right?

@navendu-pottekkat yes, that should work, ApplyManifest generates missing namespaces. Just a test away :smile: .
But maybe others have opinion about where it should be deployed (can be changed later, of course).

Reply:
The helm chart-based installation defaults to the same `meshery` namespace. Good call, @mgfeller. I second this motion.